tooling/camel-spring-boot-bom/pom.xml

```diff
@@ -916,7 +916,7 @@
       <dependency>
         <groupId>org.apache.camel.springboot</groupId>
         <artifactId>camel-jq-starter</artifactId>
-        <version>${project.version}</version>
+        <version>4.0.0</version>
```

Comment:
do you think can we use `camel-spring-boot-community.version` property instead?

Reply:
@mcarlett This is post-property resolution.